jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -59,7 +60,12 @@ def process(self, input):
                 reffile = self.get_reference_file(input_model, reftype)
                 reference_file_names[reftype] = reffile if reffile else ""
 
+            # Get the MSA metadata file if needed and add to reffiles
+            msa_metadata_file = input_model.meta.instrument.msa_metadata_file
+            if msa_metadata_file:
+                msa_metadata_file = join(self.input_dir, basename(msa_metadata_file))
```

Comment:
At points like these, the intent is to use `Step.make_input_path`, to encapsulate the appending and any other `Step` dependent things may come into play when creating paths.

Reply:
Ah, ok!  That can be fixed.